Nimble/Wrappers/ObjCMatcher.swift

```diff
@@ -118,7 +44,7 @@ typealias FullMatcherBlock = (actualExpression: Expression<NSObject>, failureMes
     }
 
     public func matches(actualBlock: () -> NSObject!, failureMessage: FailureMessage, location: SourceLocation) -> Bool {
-        let expr = Expression(expression: actualBlock, location: SourceLocation())
+        let expr = Expression(expression: actualBlock, location: location)
```

Comment:
Actually, after thinking about it more, it was a bug if the ObjC matcher was used but the passed in source location wasn't used :fearful:.

Now, I've changed the ObjCMatcher to drop location from the block, making this change mandatory.